ports/nrf/boards/feather52/board.c

```diff
@@ -35,15 +35,16 @@
 #define BOOTLOADER_VERSION_REGISTER     NRF_TIMER2->CC[0]
 uint32_t bootloaderVersion = 0;
 
-void board_init(void)
+void board_init (void)
```

Comment:
Should probably do these changes also in other board.c files. Also, is there a coding style guide for CP? I really love consistency in code so would like to got through the whole port code and make it nice and consistent :)

Reply:
I was experimenting with clang-format to standardize everything. It looks really promising. Here is where I left it off: https://github.com/tannewt/circuitpython/tree/clang-format